packages/fluentui/react-northstar/src/components/Menu/MenuDivider.tsx

```diff
@@ -9,75 +21,123 @@ import {
   ChildrenComponentProps,
   ContentComponentProps,
   rtlTextContainer,
+  ShorthandFactory,
+  ShorthandConfig,
 } from '../../utils';
-import { WithAsProp, withSafeTypeForAs, FluentComponentStaticProps, ProviderContextPrepared } from '../../types';
-import { useTelemetry, getElementType, useUnhandledProps, useAccessibility, useStyles } from '@fluentui/react-bindings';
-// @ts-ignore
-import { ThemeContext } from 'react-fela';
+import { ProviderContextPrepared } from '../../types';
 
 export interface MenuDividerProps extends UIComponentProps, ChildrenComponentProps, ContentComponentProps {
   /** Accessibility behavior if overridden by the user. */
-  accessibility?: Accessibility<never>;
+  accessibility?: Accessibility<MenuDividerBehaviorProps>;
 
-  vertical?: boolean;
-  primary?: boolean;
-  secondary?: boolean;
   inSubmenu?: boolean;
+  secondary?: boolean;
+  pills?: boolean;
+  pointing?: boolean;
+  primary?: boolean;
+  vertical?: boolean;
 }
 
-export const menuDividerClassName = 'ui-menu__divider';
-
-export type MenuDividerStylesProps = Required<Pick<MenuDividerProps, 'vertical' | 'inSubmenu' | 'primary'>> & {
+export type MenuDividerStylesProps = Required<
+  Pick<MenuDividerProps, 'vertical' | 'inSubmenu' | 'pills' | 'primary' | 'pointing'>
```

Comment:
Should we add `secondary` here? I know we are not currently using it inside the theme, but if we have the prop I believe it should be added here too..

Reply:
Yep, added